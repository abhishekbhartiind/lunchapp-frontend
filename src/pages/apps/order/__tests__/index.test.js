import React from 'react'
import renderer from 'react-test-renderer'
import { MockedProvider } from 'react-apollo/test-utils'
import Order from '../index'

it('make order', () => {
	const order = renderer
		.create(
			<MockedProvider mocks={[]}>
				<Order />
			</MockedProvider>
		)
		.toJSON()
	expect(order).toMatchSnapshot()
})
