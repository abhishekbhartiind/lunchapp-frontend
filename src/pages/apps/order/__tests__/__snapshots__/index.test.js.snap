// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`make order 1`] = `
<div
  style={
    Object {
      "marginTop": 20,
      "overflow": "hidden",
      "paddingTop": 40,
    }
  }
>
  <h2
    aria-label={null}
    className="ant-typography"
    style={
      Object {
        "WebkitLineClamp": null,
        "color": "#fff",
        "display": "block",
        "textAlign": "center",
      }
    }
  >
    Đặt món
  </h2>
  <div
    className="ant-row"
  >
    <div
      className="ant-col ant-col-22 ant-col-offset-1"
    >
      <div
        className="ant-row-flex ant-row-flex-center ant-row-flex-middle"
      >
        <div>
          Hệ thống đã khóa
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`order component match snapshot 1`] = `
<div>
  <h2 class="ant-typography" style="color: rgb(255, 255, 255); display: block; text-align: center;">
    Đặt món
  </h2>
  <div class="ant-row">
    <div class="ant-col ant-col-22 ant-col-offset-1">
      <div class="ant-list ant-list-lg ant-list-split">
        <div class="ant-spin-nested-loading">
          <div class="ant-spin-container">
            <ul class="ant-list-items">
              <li
                class="ant-list-item"
              >
                <div class="ant-list-item-meta">
                  <div class="ant-list-item-meta-content">
                    <h4 class="ant-list-item-meta-title">
                      Ga Deli
                    </h4>
                    <div class="ant-list-item-meta-description">
                      8/10
                    </div>
                  </div>
                </div>
              <div>
                3
              </div>
              <ul 
                class="ant-list-item-action"
              >
                <li>
                  <button 
                    id="minus-order-1b2cff60-a6c6-11e9-9c55-df503bf9987e"
                    type="button"
                    class="ant-btn minus-order"
                    ant-click-animating-without-extra-node="false"
                    onMouseEnter={[Function]}
                  >
                    <span>
                      -
                    </span>
                  </button>
                  <em class="ant-list-item-action-split"></em>
                </li>
                <li>
                  <button
                    id="plus-order-1b2cff60-a6c6-11e9-9c55-df503bf9987e"
                    type="button"
                    class="ant-btn plus-order"
                    ant-click-animating-without-extra-node="false"
                    onMouseEnter={[Function]}
                  >
                    <span>
                      +
                    </span>
                  </button>
                </li>
              </ul>
              <button
                id="note-order-1b2cff60-a6c6-11e9-9c55-df503bf9987e"
                type="button"
                class="ant-btn ant-btn-primary"
                ant-click-animating-without-extra-node="false"
                onMouseEnter={[Function]}
              >
                <span>
                  Note
                </span>
              </button>
              </li>
              <li class="ant-list-item" style="background-color: rgb(255, 255, 255); margin-bottom: 20px; padding: 20px;">
                <div class="ant-list-item-meta">
                  <div class="ant-list-item-meta-content">
                    <h4 class="ant-list-item-meta-title">
                      Ca Deli
                    </h4>
                    <div class="ant-list-item-meta-description">
                      0/10
                    </div>
                  </div>
                </div>
                <div>
                  0
                </div>
                <ul class="ant-list-item-action">
                  <li>
                    <button
                      id="minus-order-a500f6b0-a6c6-11e9-9c55-df503bf9987e"
                      disabled=""
                      type="button"
                      class="ant-btn minus-order"
                      onMouseEnter={[Function]}
                    >
                      <span>
                        -
                      </span>
                    </button>
                    <em class="ant-list-item-action-split"></em>
                  </li>
                  <li>
                    <button
                      id="plus-order-a500f6b0-a6c6-11e9-9c55-df503bf9987e"
                      type="button"
                      class="ant-btn plus-order"
                      onMouseEnter={[Function]}
                    >
                      <span>
                        +
                      </span>
                    </button>
                  </li>
                </ul>
                <button
                  id="note-order-a500f6b0-a6c6-11e9-9c55-df503bf9987e"
                  type="button"
                  class="ant-btn ant-btn-primary"
                  disabled=""
                  onMouseEnter={[Function]}
                >
                  <span>Note</span>
                </button>
              </li>
              <li class="ant-list-item" style="background-color: rgb(255, 255, 255); margin-bottom: 20px; padding: 20px;">
                <div class="ant-list-item-meta">
                  <div class="ant-list-item-meta-content">
                    <h4 class="ant-list-item-meta-title">
                      Tom Deli
                    </h4>
                    <div class="ant-list-item-meta-description">
                      1/15
                    </div>
                  </div>
                </div>
                <div>
                  1
                </div>
                <ul class="ant-list-item-action">
                  <li>
                    <button
                      id="minus-order-a976a050-a6c6-11e9-9c55-df503bf9987e"
                      type="button"
                      class="ant-btn minus-order"
                      onMouseEnter={[Function]}
                    >
                      <span>
                        -
                      </span>
                    </button>
                    <em class="ant-list-item-action-split"></em>
                  </li>
                  <li>
                    <button
                      id="plus-order-a976a050-a6c6-11e9-9c55-df503bf9987e"
                      type="button"
                      class="ant-btn
                      plus-order"
                      onMouseEnter={[Function]}
                    >
                      <span>
                        +
                      </span>
                    </button>
                  </li>
                </ul>
                <button
                  id="note-order-a976a050-a6c6-11e9-9c55-df503bf9987e"
                  type="button"
                  class="ant-btn
                  ant-btn-primary"
                  onMouseEnter={[Function]}
                >
                  <span>
                    Note
                  </span>
                </button>
              </li>
            </ul>
          </div>
        </div>
      </div>
      <div class="ant-row-flex ant-row-flex-center ant-row-flex-bottom"></div>
    </div>
  </div>
</div>
`;
